fe/fe-core/src/main/java/org/apache/doris/mysql/privilege/UserProperty.java

```diff
@@ -574,6 +601,14 @@ public void write(DataOutput out) throws IOException {
             entry.getValue().write(out);
         }
 
+        // default cloud cluster
+        if (defaultCloudCluster == null) {
```

Comment:
This is an incompatible change and should be carefully changed.
The proposed impl. seems not correct.

Reply:
consider using gson from this PR on